resources/web/README.md

```diff
@@ -1,15 +1,69 @@
+---    
+sidebar: auto
+---      
+
 # Web Development
 
```

Comment:
An overview of web development or what the web is would be helpful here. We go straight into resources for FrontEnd vs Backend.